packages/changesets/README.md

```diff
@@ -0,0 +1,83 @@
+# Changesets
+
+To make sure that the adopters of our products and integrations can always use the latest and greaters versions we implemented a standarization for Release Notes and Migration Guides. The goal is to make the migration process to the newer version of each package as smooth and effortless as possible.
+
+To facilitate the process of creating the release notes and migration guides we use the [changesets](https://github.com/changesets/changesets), and `@vue-storefront/changesets` that provides a custom changelog format for the `changesets` package.
+
+## Installation
+
+```sh
+yarn add -D @vue-storefront/changesets
+```
+
+## Usage
+
+Add the following to your `changeset/config.json` file:
+
+```json[.changeset/config.json]
+{
+  "changelog": "@vue-storefront/changesets",
+}
+```
+
+This will use the `@vue-storefront/changesets` package to [modify the changelog format](https://github.com/changesets/changesets/blob/main/docs/modifying-changelog-format.md). 
+
+## Example
+
+1. [Use the `changeset` CLI to create a new changeset](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md#i-am-in-a-multi-package-repository-a-mono-repo)
+2. Provide a changeset, following the [conventions](#conventions) eg. `[REMOVED] getPhysicalStores method was removed from the Middleware and SDK.`
+3. Provide another changeset eg. `[ADDED] getPhysicalStores method that allows fetching a list of physical stores and filtering it by their location.`
+4. Merge changes to the main branch
+
+Changeset will create a PR for relase, and the following markdown:
+
+```md
+[REMOVED] getPhysicalStores method was removed from the Middleware and SDK.
+[ADDED] getPhysicalStores method that allows fetching a list of physical stores and filtering it by their location.
+```
+
+will be transformed into:
+
+```md
+- **[REMOVED]** getPhysicalStores method was removed from the Middleware and SDK.
+- **[ADDED]** getPhysicalStores method that allows fetching a list of physical stores and filtering it by their location.
+```
+
+In the changeset's release PR makes sure, that the migration guide is provided in the CHANGELOG.md file.
+
+## Conventions
+
+- [ADDED] { what was added, with link to the docs }, { what new capabilities it introduced }{ example usage }
+- [CHANGED] { what has changed }, { why it has changed}{ before / after code }
+- [FIXED] { what was fixed }, { what was the previous behaviour}. { what is the new behaviour }
+- [REMOVED] { what was removed }, { why }. { additional context for ones using this feature eg. it is now handled in a different way in a different place }
```

Comment:
e.g. dependency upgrade with breaking changes should fall under "changed"?

Reply:
for the deps update, changesets creates the changelog automatically 
https://github.com/vuestorefront/vue-storefront/blob/851e7b0dcbc85013592edbddd247e456e6d1e7b6/packages/changesets/src/__tests__/changelog.spec.ts#L67